### PR TITLE
[stable/rabbitmq] Fix auto-generated NodePort

### LINKED ISSUE #7909: [stable/rabbitmq] Random NodePort configuration unavaliable
**Is this a request for help?**:
request
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT 

**Version of Helm and Kubernetes**:
```
helm version
Client: &version.Version{SemVer:"v2.10.0", GitCommit:"9ad53aac42165a5fadc6c87be0dea6b115f93090", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.10.0", GitCommit:"9ad53aac42165a5fadc6c87be0dea6b115f93090", GitTreeState:"clean"}
kubectl version
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.3", GitCommit:"2bba0127d85d5a46ab4b778548be28623b32d0b0", GitTreeState:"clean", BuildDate:"2018-05-21T09:17:39Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T20:08:34Z", GoVersion:"go1.10.2", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:
stable/rabbitmq

**What happened**:
Cannot user random port for the NodePort service type.
`spec.ports[1].nodePort: Invalid value: 5672: provided port is not in the valid range. The range of valid ports is 30000-32767`

**What you expected to happen**:
1) Specify `serviceType=NodePort`
2) Do not specify `rabbitmq.nodePort`
3) nodePort for amqp port chosen by kubernetes and accessible outside of the cluster

**How to reproduce it** (as minimally and precisely as possible):
`helm install --set serviceType=NodePort stable/rabbitmq`

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by ElijahReva**
Signed-off-by: Ilya Reva <ilya.reva@gmail.com>

#### What this PR does / why we need it:
This is a bug fix.
https://github.com/helm/charts/issues/7909

#### Which issue this PR fixes
    - fixes #7909 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: rabbitmq
-version: 3.0.1
+version: 3.0.2
```
Comment (juan131) [inline stable/rabbitmq/Chart.yaml]: Please increase minor version instead.

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for the PR!! Could you please adapt the README.md file to reflect these changes?

File: stable/rabbitmq/Chart.yaml (hunk 1 above)
Comment (ElijahReva) [reply stable/rabbitmq/Chart.yaml]: Updated to 3.2.0

**Comment by ElijahReva** [comment]
@juan131 Thanks for your review!
I've updated the readme with new values.

**Review by juan131 — APPROVED** [review]
Great!! Thanks for the quick update!

**Pull request merged**